### PR TITLE
refactor: port ReportCard to TS/SWR

### PR DESCRIPTION
**Pull request by olav**

### CODE REVIEW

File: src/interfaces/project.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export interface IProjectHealthReport extends IProject {
     staleCount: number;
     potentiallyStaleCount: number;
     activeCount: number;
-    updatedAt: Date;
+    updatedAt: string;
```
Comment (ykhedher) [inline src/interfaces/project.ts:27]: is it really a type of string ?

**Review by ykhedher — APPROVED** [review]
Looks good for me.

File: src/interfaces/project.ts (hunk 1 above)
Comment (olav) [reply src/interfaces/project.ts:27]: Yep! This type describes what we get from the backend.

**Review by FredrikOseberg — APPROVED** [review]
LGTM

**Pull request merged**